Kitodo/src/main/java/org/kitodo/production/forms/CurrentTaskForm.java

```diff
@@ -403,13 +405,28 @@ public void generateMissingImages() {
      *            message displayed to the user (key for resourcebundle)
      */
     private void generateImages(GenerationMode mode, String messageKey) {
+        Folder generatorSource = myProcess.getProject().getGeneratorSource();
+        List<Folder> contentFolders = currentTask.getContentFolders();
+        if (Objects.isNull(generatorSource)) {
+            Helper.setErrorMessage("noSourceFolderConfiguredInProject");
+        }
+        if(Objects.isNull(contentFolders)){
+            Helper.setErrorMessage("noImageFolderConfiguredInProject");
+        }
```

Comment:
Shouldn't we leave the method if either of these conditions is met, instead of executing the `try` block?